include/sqlite_orm/sqlite_orm.h

```diff
@@ -7186,6 +7186,12 @@ namespace sqlite_orm {
 
             prepared_statement_t(T t_, sqlite3_stmt *stmt_, connection_ref con_) :
                 prepared_statement_base{stmt_, std::move(con_)}, t(std::move(t_)) {}
+
```

Comment:
please add this code into `dev/prepared_statement.h` file also. It is better to add new code inside `dev` sources and then call amalgamate script. To run amalgamate script use this command:
```
python third_party/amalgamate/amalgamate.py -c third_party/amalgamate/config.json -s .
```